test/cypress/integration/widgets/user_groups_spec.ts

```diff
@@ -1,8 +1,19 @@
 // @ts-nocheck File not migrated fully to TS
 describe('User group management widget', () => {
     const groupName = 'user_groups_test';
+    const ldapGroupColumnName = 'LDAP group';
+    const userGroupsWidgetName = 'userGroups';
+    const setLdapAvailibility = (isEnabled: boolean) => {
+        const ldapResponse = isEnabled ? 'enabled' : 'disabled';
+        cy.intercept('GET', '/console/sp/ldap', ldapResponse);
+    };
+    const reloadPage = () => {
+        cy.usePageMock(userGroupsWidgetName).mockLogin();
+    };
```

Comment:
1. You can make it to be one-liner 🙂 
```suggestion
    const reloadPage = () => cy.usePageMock(userGroupsWidgetName).mockLogin();
```

2. The name of the function suggests that we just reload a page, but from the implementation it seems that we are logging in to the application. 
3. Maybe just `cy.reload` would be sufficient for our case? (assuming that `GET /console/sp/ldap` is called on page reload)

Reply:
1. Yup, but I don't think, that it makes an impact there
And in my opinion this approach may be more suitable, because when you're looking at the name of the variable `reloadPage`, you are able to see the part of the implementation immediately (because it's under each other)

2. Yeah, but I didn't have thought about the better name, but I'll try to figure that out once more. If you'd have any suggestion about the better name, please let mne know.

As I'm thinking about this, I think that the name `reloadPage` it's not that bad, because when it comes to it's usage in tests scenarios, we don't want to know how the "reload" is being implemented, but we just want to somehow refresh the page.

Additionally, from the tests and PR perspective, when we would add some modyfication for the "refresh" mechanism, then we can do that without having to rename all of the occurences of that function.

So eg. we want to change the "refresh" mechanism, so that it may work as you've described in the 3rd point.
Then we just modify the body of that method, without having an impact on the other parts of the application (in that case, the "other parts" would be the tests scenario), because from their perspective, they want the page to be refresh/reloaded, when the method is being invoked but the implementation of that functionality is totally transparent to them.

So we might think about that like some sort of dependency injection 😁

Let me know what you think about that

3. I tried simple `cy.reload`, but I didn't get the mocked pages, but now as I'm looking at it, I should be able to just add `usePageMock`, so that the reloaded page provides me with the pages that I need 😁

Reply:
`cy.reload` works without a problem, so I've included it with the other changes :)

Reply:
Great, thanks 🙂